src/components/touchables/GenericTouchable.tsx

```diff
@@ -214,14 +218,15 @@ export default class GenericTouchable extends Component {
       this.handleGoToUndetermined();
       if (shouldCallOnPress) {
         // Calls only inside component whether no long press was called previously
-        this.props.onPress && this.props.onPress();
+        this.props.onPress?.();
       }
     }
   };
 
   onLongPressDetected = () => {
     this.longPressDetected = true;
-    this.props.onLongPress();
+    // checked for in the caller of `onLongPressDetected`, but better to check twice
+    this.props.onLongPress?.();
```

Comment:
I'm not sure about that one.

Reply:
What do you mean? I think it was wrong that there was no check before and TS does not allow it without `?.` too.

Reply:
Sorry, I made really ambiguous comments that day 😄 

I think I referred to the comment, We should rewrite it in the cleaner manner, so TS can infer if it can be undefined or not.

Reply:
Not sure if it is possible, this function could be used somewhere else too, and there could be no check. I think it is safe to leave as it is now.